### ISSUE TITLE
Measurement Coverage by Test Class - Improve graph and how info is communicated

### ISSUE THREAD

**Issue by agrabeli** [ooni/explorer#123]
I find the "Measurement Coverage by Test Class" graph (on country pages) quite confusing, because even though it says (in the pop-up window) that a certain amount of tests have been run (e.g. performance tests), this is not displayed in the graph itself. 

For example, in Benin (http://explorer-beta.ooni.io/country/BJ), when I hover over February 2019, it says that "5 performance tests" have been run, yet this is not displayed in the graph. 

Furthermore, the information provided via the pop-up window (which appears when you hover over the graph) is too cluttered (in some cases, you can't read everything):

![measurement-coverage](https://user-images.githubusercontent.com/8290574/57095330-e0363a80-6d12-11e9-96d1-e0df45fdcae1.png)

What do we mean by "networks"? If we're displaying the volume of measurements per test class, why are we displaying the amount of networks too?

**Comment by agrabeli** [comment]
I guess this is also discussed here: https://github.com/ooni/explorer/issues/90

**Comment by hellais** [comment]
> For example, in Benin (http://explorer-beta.ooni.io/country/BJ), when I hover over February 2019, it says that "5 performance tests" have been run, yet this is not displayed in the graph.

Yes this definitely looks like a bug. I filed a separate issue for it here: https://github.com/ooni/explorer/issues/217

> What do we mean by "networks"? If we're displaying the volume of measurements per test class, why are we displaying the amount of networks too?

I think these are more substantial questions about how to improve this chart in the country page.

The rational behind doing it this way was that we wanted to provide a chart that gives an idea to the user "how well covered" a particular country is. The way to estimate that is to look at the volume of measurement over time and the number of networks tested over time.

Maybe this is not the best way to determine "how well covered" a country is, but I suggest we postpone this discussion to after the stable launch, as we made this decision pretty early and put quite some though into it.

I am removing the bug label and putting this issue back in the icebox.

**Comment by hellais** [comment]
We discussed this during the team meeting too. Here are the notes from the discussion:

Replace the measurement per test class coverage chart (in country pages) with a better and more concise way of communicating if we have good measurement coverage per test class in that country => Federico (issue in ooni/backend)
